libs/ledgerjs/packages/cryptoassets/src/crypto-assets-importer/exchange/bep20.ts

```diff
@@ -6,15 +6,21 @@ type BEP20Exchange = [string, string, string];
 export const importBEP20Exchange = async (outputDir: string) => {
   console.log("importing bep 20 exchange tokens...");
   try {
-    const bep20Exchange = await fetchTokens<BEP20Exchange>("exchange/bep20.json");
+    const [bep20Exchange, hash] = await fetchTokens<BEP20Exchange>("exchange/bep20.json");
     fs.writeFileSync(`${outputDir}/exchange/bep20.json`, JSON.stringify(bep20Exchange));
+    if (hash) {
```

Comment:
Maybe we should name it for what it is eg. etag ? more than being a hash it's an etag representing a version of content

I find it similar to naming a string variable "str"

Reply:
An etag is a hash, so I don't think it's really an issue.
+ this hash will be changed from an ETag to a commit hash once we migrate to the CAL service

Reply:
versionHash ?